aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -120,6 +114,14 @@ template <class T> Vec256<T> operator+(const Vec256<T> &a, const Vec256<T> &b) {
   return c;
 }
 
+template <class T> Vec256<T> operator-(const Vec256<T> &a, const Vec256<T> &b) {
+  Vec256<T> c = Vec256<T>();
+  for (int i = 0; i != c.size; i++) {
```

Comment:
this is a bit misleading because `.size` is really `Vec256<T>::size`. Can we use the later to make it clear that it's a static member?